kinto/views/__init__.py

```diff
@@ -29,4 +30,10 @@ def object_exists_or_404(request, collection_id, object_id, parent_id=''):
                            object_id=object_id)
     except exceptions.RecordNotFoundError:
         # XXX: We gave up putting details about parent id here (See #53).
-        raise httpexceptions.HTTPNotFound()
+        details = {
+                    "id": object_id,
+                    "resource_name": collection_id
+                }
```

Comment:
The indentation should be like the one you did in  ` kinto/core/resource/__init__.py `